gammapy/maps/core.py

```diff
@@ -1111,6 +1111,38 @@ def _plot_interactive(**ikwargs):
                 fig, ax, cbar = img.plot(stretch=stretch, **kwargs)
                 plt.show()
 
+    def plot_mask(self, ax=None, **kwargs):
+        """Plot the mask as a shaded area
+
+        Parameters
+        ----------
+        ax : `~matplotlib.axis`
+            Axis instance to be used for the plot.
+        **kwargs : dict
+            Keyword arguments passed to `~matplotlib.pyplot.axvspan`
+
+        Returns
+        -------
+        ax : `~matplotlib.pyplot.Axis`
+            Axis used for plotting
+        """
+        import matplotlib.pyplot as plt
+
+        if not self.is_mask:
+            raise ValueError("This is not a mask and cannot be plotted")
+
+        kwargs.setdefault("alpha", 0.05)
+        kwargs.setdefault("colors", "k")
+
+        ax = plt.gca() if ax is None else ax
+
+        data = self.data.astype(float)
+
+        cs = ax.contourf(data, level=[0, 0.5], **kwargs)
+        ax.contour(cs)
```

Comment:
I think for now there is no need to call `ax.contour()` in addition, so no extra plotting of the boundary...